docs/start/authentication/graphql-storefront.mdx

```diff
@@ -49,7 +49,8 @@ content-type: application/json
   "expires_at": 1602288000,   // when the token will expire, as an integer unix timestamp (in seconds)
   "allowed_cors_origins": [   // array of origins (up to 2 origins per token)
     "https://example.com"
-  ]
+  ],
+  "dev_mode": true
```

Comment:
not sure if we want to include it here in the examples, we might want a completely separate section